sound/soc/sof/core.c

```diff
@@ -96,7 +96,7 @@ struct snd_sof_pcm *snd_sof_find_spcm_pcm_id(struct snd_sof_dev *sdev,
 }
 
 struct snd_sof_widget *snd_sof_find_swidget(struct snd_sof_dev *sdev,
-					    char *name)
+					    const char *name)
 {
```

Comment:
this is a different patch, send it in a different PR

Reply:
ok, I will refine it